.gitignore

```diff
@@ -1,18 +1,18 @@
-# Node modules #
-/node_modules
+/node_modules/
+/.idea/
+/doc/
+/coverage/
+/site
+/report/
+/.c9revisions/
+/.settings/
 
-# Logs #
-/log
-
-# Temporal directory #
-/temp
-
-# Database #
-/data
-
-# Specific files #
+.settings
+.classpath
+.project
+.metadata
 npm-debug.log
 
-# JetBrains IDEs #
-## Directory-based project format:
-.idea/
+/data/
+/log/
```

Comment:
- `data` is typically used (in fact, `data/db` as the default directory where Mongo stores its files) and since to run the tests a Mongo instance is needed, I typically create that directory in the root of the project
- `temp` is where the generated CVS files are stored before serving them to the portal
- `log` is the default directory where log files are stored
Those entries are just to avoid any of the files in these directories to land in the repo... ;)